src/components/MessageList/VirtualizedMessageList.js

```diff
@@ -32,91 +29,117 @@ const VirtualizedMessageList = ({
   hasMore,
   loadingMore,
   messageLimit = 100,
-  overscan = 200,
+  overscan = 0,
   shouldGroupByUser = false,
   customMessageRenderer,
+  // TODO: refactor to scrollSeekPlaceHolderConfiguration and components.ScrollSeekPlaceholder, like the Virtuoso Component
   scrollSeekPlaceHolder,
-  Message = FixedHeightMessage,
+  Message = DefaultMessage,
```

Comment:
Seeing that a fixed height message might not be required anymore, is it possible to use the `MessageTeam` component in the virtualized list now?

Looks like this Message property requires a type with `FixedHeightMessageProps` still, so not sure how to do it...

Reply:
@petyosi can you speak to the feasibility of using our other Message UI components in the VL now?

Reply:
@kainosnoema indeed, using "any" messages in the virtual list is something which we are looking into, but not yet ready to introduce as the default configuration. The part which does not work as smoothly as we wish is messages which contain `img` elements without a specified size. Such posts may cause the virtual list to "jump" when the image is loaded.

This being said, you can ignore the typing for now and try passing the plain message - we would love to get your feedback on the performance of that.

Reply:
@petyosi thanks for the update. We're building our own custom message component and will be having photo galleries with a fixed height (when attachments are present), so don't think we'll run into the jumping issue. We'll give this a shot!